packages/lib/src/vis/tiles/TiledLayer.tsx

```diff
@@ -20,23 +20,29 @@ function TiledLayer(props: Props) {
   const { baseLayerIndex, numLayers, tileSize } = api;
   const layerSize = api.layerSizes[layer];
 
-  const { visSize } = useAxisSystemContext();
+  const { abscissaConfig, ordinateConfig, visSize } = useAxisSystemContext();
   const { xVisibleDomain, yVisibleDomain } = useScaledVisibleDomains(layerSize);
 
-  // Transform visible domain to current level-of-detail array coordinates
   const tileOffsets = getTileOffsets(xVisibleDomain, yVisibleDomain, tileSize);
 
   // Sort tiles from closest to vis center to farthest away
   const center = new Vector2(sum(xVisibleDomain) / 2, sum(yVisibleDomain) / 2);
   sortTilesByDistanceTo(tileOffsets, tileSize, center);
 
+  const abscissaFlip = abscissaConfig.flip ? -1 : 1;
+  const ordinateFlip = ordinateConfig.flip ? -1 : 1;
+
   return (
-    // Tranforms to use level of details layer array coordinates
+    // Tranforms to handle axes flip and use level of details layer array coordinates
     <group
-      position={[-visSize.width / 2, -visSize.height / 2, layer / numLayers]}
+      position={[
+        abscissaFlip * (-visSize.width / 2),
+        ordinateFlip * (-visSize.height / 2),
+        layer / numLayers,
+      ]}
       scale={[
-        visSize.width / layerSize.width,
-        visSize.height / layerSize.height,
+        (abscissaFlip * visSize.width) / layerSize.width,
+        (ordinateFlip * visSize.height) / layerSize.height,
```

Comment:
That's what I suspected: the flip must be applied on both the position and the scale.

In this case, I think it makes more sense to have a separated group that flips everything around as you did in the first place. Sorry !